docs/deployment.md

```diff
@@ -66,6 +66,10 @@ Options:
                                   Enable/Disable X-Forwarded-Proto,
                                   X-Forwarded-For, X-Forwarded-Port to
                                   populate remote address info.
+  --default-server-header / --no-default-server-header
```

Comment:
maybe a better name would be `enable-server-header / -- disable-server-header` ? when I read `default-server-header` I would like to see about an option to set it when here it's just a switch.
same for date ?
thoughts ?

Reply:
Naming is tough ;)

I chose these names because they reflect the behaviour as I understand it (but my understanding may be imperfect): if the headers are not provided by the application, uvicorn has default values for these headers, unless you disable that default through this new feature. Users may read `--disable-server-header` as "uvicorn will never send a Server header".

Do `--disable-default-server-header` and `--disable-default-date-header` make more sense?

Reply:
> Do --disable-default-server-header and --disable-default-date-header make more sense?

I think that's a good compromise between the 2, @Kludex any preference ? 
I'd prefer having more eyes on this : as you said naming is tough and one can see different things for the same variable name ; so the more eyes will have the better, until everyone disagree :)
let's wait a little bit on this.

Reply:
I'm more in favor of following the pattern already in place. I mean, the option above is `--proxy-headers / --no-proxy-headers", so I guess `--server-header / --no-server-header` makes more sense to me. What do you guys think?

Reply:
Fair point. Disregard my `--disable-*` suggestion, I agree that having a predictable interface is more important than having a concise one.

Reply:
ok let's go for that last suggestion !